tools/tsp-client/src/options.ts

```diff
@@ -85,19 +72,6 @@ export async function getOptions(): Promise<Options> {
     process.exit(0);
   }
 
-  if (values.emitter) {
-    let emitter = values.emitter.toLowerCase();
-    if (emitter in languageAliases) {
-      emitter = languageAliases[emitter]!;
-    }
-    if (!(knownLanguages as readonly string[]).includes(emitter)) {
```

Comment:
assuming this gets somewhere else, do we still need the `languageSettings.js` file at all or can we remove it?

Reply:
Removed!